modules/account-lib/src/coin/hbar/keyPair.ts

```diff
@@ -0,0 +1,22 @@
+import { Ed25519KeyPair } from '../baseCoin/ed25519KeyPair';
+import { KeyPairOptions, ByteKeys } from '../baseCoin/iface';
+
+export class KeyPair extends Ed25519KeyPair {
+  /**
+   * Public constructor. By default, creates a key pair with a random master seed.
+   *
+   * @param { KeyPairOptions } source Either a master seed, a private key (extended or raw), or a public key
+   */
+  constructor(source?: KeyPairOptions) {
+    super(source);
+  }
+
+  /**
+   * Ethereum default keys format is raw private and uncompressed public key

```

Comment:
nit: update comment

Reply:
Great, I've created a new PR for keypair: https://github.com/BitGo/BitGoJS/pull/807
And I am going to be working on these comments there.